pkg/schedulingtypes/replicascheduler.go

```diff
@@ -190,6 +191,34 @@ func (s *ReplicaScheduler) Reconcile(obj pkgruntime.Object, qualifiedName ctluti
 	}
 
 	key := qualifiedName.String()
+
+	if rsp.Spec.PlacementIntersection {
+		klog.V(3).Infof("Computing placement of resource %q", qualifiedName)
+
+		fedClusters, err := s.podInformer.GetReadyClusters()
+		if err != nil {
+			runtime.HandleError(errors.Wrapf(err, "Failed to get KubefedCluster while reconciling RSP named %q", key))
+			return ctlutil.StatusError
+		}
+
+		resultClusters, err := plugin.(*Plugin).GetResourceClusters(qualifiedName, fedClusters)
+		if err != nil {
+			runtime.HandleError(errors.Wrapf(err, "Failed to get prefrerred clusters while reconciling RSP named %q", key))
+			return ctlutil.StatusError
+		}
+
+		preferredClusters := []string{}
```

Comment:
Could not understand the need of `preferredClusters` it seems to be a copy of `resultClusters`

Reply:
nit: @xunpan this is still unresolved but I will approve as this does not seem to harm the logic at all.